omnigan/generator.py

```diff
@@ -228,8 +229,8 @@ def __init__(self, opts):
         super().__init__(
             opts.gen.s.n_upsample,
             opts.gen.s.n_res,
-            opts.gen.s.res_dim,
-            opts.gen.s.output_dim,
+            dim=opts.gen.s.res_dim,
```

Comment:
`proj_dim`?

Reply:
Haha I would eventually have changed that once I started implementing the segmentation head